lib/std/start.zig

```diff
@@ -176,7 +176,7 @@ fn _DllMainCRTStartup(
     lpReserved: std.os.windows.LPVOID,
 ) callconv(std.os.windows.WINAPI) std.os.windows.BOOL {
     if (!builtin.single_threaded and !builtin.link_libc) {
-        _ = @import("start_windows_tls.zig");
+        _ = @import("os/windows/tls.zig");
```

Comment:
I guess it's good that you didn't call it start_tls.zig :stuck_out_tongue_winking_eye: